packages/preview-middleware/src/base/flp.ts

```diff
@@ -192,7 +194,7 @@ export class FlpSandbox {
         this.templateConfig = {
             basePath: posix.relative(posix.dirname(this.config.path), '/') ?? '.',
             apps: {},
-            customInit: this.config.customInit,
+            customInit: this.config.init,
```

Comment:
As `customInit` is being renamed everywhere: Shouldn't this be `init: this.config.init,` here as well (and adjusted in the `TemplateConfig` [interface](https://github.com/SAP/open-ux-tools/blob/a68d213f9da984949d9d5594acfc75060c9d1884/packages/preview-middleware/src/base/flp.ts#L130))? Additionally the `TemplateConfig` interface contains a `test` property [leftover](https://github.com/SAP/open-ux-tools/blob/a68d213f9da984949d9d5594acfc75060c9d1884/packages/preview-middleware/src/base/flp.ts#L125) from my initial changes. I think that one can be deleted to avoid confusion.